stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/RemoteMessageExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package io.getstream.chat.android.client.extensions
+
+import com.google.firebase.messaging.RemoteMessage
+import io.getstream.chat.android.client.notifications.handler.NotificationConfig
+
+public fun RemoteMessage.isValid(notificationConfig: NotificationConfig): Boolean =
```

Comment:
Do we want to provide this as separate public API, shouldn't checks just go through `ChatClient`'s companion?

Having gone through all the code on this PR, it looks like we have... Five (?) public methods for checking if a message is valid now. Do we really need to expose all of these?